wordpress-mobile/WordPress-iOS
# Jetpack Section: Scan - Push to the threat details view

Project #15190
---

This also hides the fix / ignore buttons for fixed / ignored threats. 

### To test:
1. Launch the app
2. Tap the My Site view
3. Tap on a site with Jetpack Scan enabled
4. Tap on Scan
5. Tap on History
6. Tap on a threat
7. You should be pushed to the threat details view
8. The fix / ignore buttons should not be visible

### PR submission checklist:
